### PR TITLE
[ADD]website_snippet_timeline: Backport to 13.0

### PR DESCRIPTION
**Pull request by willytal**
TT36614

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Functionally tested

File: website_snippet_timeline/static/src/js/options.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    options.registry.Timeline = options.Class.extend({
+        // --------------------------------------------------------------------------
+        // Options
+        // --------------------------------------------------------------------------
+
+        /**
+         * Moves the card to the right/left.
+         *
+         * @see this.selectClass for parameters
+         */
```
Comment (Tardo) [inline website_snippet_timeline/static/src/js/options.js]:
```suggestion
        /**
         * @override
         */
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Please, update the comment. Tested on runboat and works fine... It has a small problem when you try to move to the left in a row with cards on both sides.

**Review by tarteo — APPROVED** [review]
Tested on runboat. LGTM!

**Comment by Tardo** [comment]
Please, checks the behavior when there are cards in both columns:
timeline

**Review by Tardo — APPROVED** [review]
Thanks +1

**Pull request merged**